src/components/ImportExport.js

```diff
@@ -181,6 +183,47 @@ class ImportExport extends React.Component {
     }
   }
 
+  generateCSV = () => {
+    const { hydrants, trails } = this.props;
+    const hydrantsRows = [
+      ['Trail_Name', 'Hyd_Index', 'Hyd_ID', 'Hyd_State', 'Hyd_Hours',
+      'Hyd_Gun', 'Hyd_Gpm', 'Hyd_Notes', 'Hyd_Elevation',
+      'Hyd_Target_Gallons', 'Hyd_Total_Gallons', 'Hyd_Cfm', 'Hyd_Pressure_Zone']
+    ]
+    trails.keySeq().forEach((trailId) => {
+      const trailName = trails.get(trailId).get('name')
```

Comment:
This needs to be the old format trail name for this db schema. 

`trailName = trails.getIn([trailId, 'features', 0].get('description').split(',')[0]`